ocean_lib/models/arguments.py

```diff
@@ -6,6 +6,8 @@
 import warnings
 from typing import Any, Dict, List, Optional
 
+from web3.main import Web3
```

Comment:
There should be simple unit tests for each class in this module. Eg via a test module test_arguments.py